src/test/java/org/sagebionetworks/bridge/models/DateTimeHolderTest.java

```diff
@@ -0,0 +1,25 @@
+package org.sagebionetworks.bridge.models;
+
+import static org.sagebionetworks.bridge.TestConstants.TIMESTAMP;
+import static org.testng.Assert.assertEquals;
+
+import org.sagebionetworks.bridge.json.BridgeObjectMapper;
+import org.testng.annotations.Test;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+public class DateTimeHolderTest {
+    
+    @Test
+    public void canSerialize() throws Exception {
+        DateTimeHolder holder = new DateTimeHolder(TIMESTAMP);
+        
+        JsonNode node = BridgeObjectMapper.get().valueToTree(holder);
+        
+        assertEquals(TIMESTAMP.toString(), node.get("dateTime").textValue());
+        assertEquals("DateTimeHolder", node.get("type").textValue());
+        
+        DateTimeHolder deser = BridgeObjectMapper.get().readValue(node.toString(), DateTimeHolder.class);
+        assertEquals(deser.getDateTime(), TIMESTAMP);
```

Comment:
Similarly, DateTime.equals() doesn't check timestamp. We want to verify the timezone is correct. I believe TestUtils.assertDatesWithTimeZoneEqual() will do this for you.